metadata-ingestion/src/datahub/emitter/mce_builder.py

```diff
@@ -132,7 +145,11 @@ def dataset_key_to_urn(key: DatasetKeyClass) -> str:
     )
 
 
-def make_container_urn(guid: str) -> str:
+def make_container_urn(guid: Union[str, "DatahubKey"]) -> str:
```

Comment:
Is the idea that you must be using one of these concrete classes like DatabaseKey etc. that extend DataHubKey?
So each container entity type must create a concrete Key class?

Reply:
effectively yes - we already have concrete key classes across our sources, and I much prefer the real classes over passing a property bag dict around everywhere